### PR TITLE
Update dont pre select values in the pupil registration #1259

### PR DESCRIPTION
**Pull request by cr-eative-dev**
### What was done:

- Changed initial state in registration context so there is no pre-selected value for schoolGrade (Schulklasse), schoolType (Schulform) and userState (Schulort)
- added additional checks in the registration context and also in the individual components before the onNext function is called to make sure the user selects their answer before continuing in the registration flow
- a hint asking the user to make a selection before continuing is shown in case there is no selection made, when a selection is made the hint disappears again
- the user is no longer able to just click through the registration flow without deliberatly selecting an option
- added translations for german, english, turkish, arabic, ukrainian and russian

### Closes Ticket:
https://github.com/corona-school/project-user/issues/1259

### CODE REVIEW

**Review by JeangelLF — COMMENTED** [review]
Very nice 🙌! Thank you for your PR

I just left a few minor comments, let me know if you have any questions.

File: src/pages/registration/SchoolClass.tsx [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
     const handleOnSchoolClassChange = (newClass: number) => {
         setSchoolClass(newClass);
         if (!schoolType) {
             setSchoolType(newClass === TRAINEE_GRADE ? 'berufsschule' : 'grundschule');
         }
     };
 
+    const handleNext = () => {
+        if (!schoolClass) {
+            setShowClassMissing(true);
```
[the remark this answers is not in the document]
Comment (cr-eative-dev) [reply src/pages/registration/SchoolClass.tsx:26]: I added logic to take care of the issue

File: src/pages/registration/UserState.tsx [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
 
 const UserState: React.FC = () => {
     const { userState, setUserState, onPrev, onNext } = useContext(RegistrationContext);
     const { space } = useTheme();
     const { t } = useTranslation();
     usePageTitle('Lern-Fair - Registrierung: Schulort');
 
+    const [showStateMissing, setShowStateMissing] = useState<boolean>(false);
+
+    const handleNext = () => {
```
[the remark this answers is not in the document]
Comment (cr-eative-dev) [reply src/pages/registration/UserState.tsx:18]: I added logic to take care of the issue

File: src/pages/registration/SchoolClass.tsx [hunk 3]
```diff
@@ -1,29 +1,41 @@
...
+        }
+    };
+
     return (
         <VStack flex="1" marginTop={space['1']}>
             <Heading>{t(`registration.steps.grade.subtitle`)}</Heading>
             <Row flexWrap="wrap" w="100%" mt={space['1']} marginBottom={space['1']}>
                 <GradeSelector grade={schoolClass} onGradeChange={handleOnSchoolClassChange} />
             </Row>
+            {showClassMissing && <AlertMessage content={t('registration.hint.schoolClassSelectionMissing')} />}
```
[the remark this answers is not in the document]
Comment (cr-eative-dev) [reply src/pages/registration/SchoolClass.tsx:38]: I added logic and translations so we have the same behaviour also in the school type component

**Review by JeangelLF — APPROVED** [review]
Nice 💯, Thank you!

**Pull request merged**